src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -163,8 +170,17 @@ class PillSelectorDropdown extends React.Component<Props, State> {
         this.handleInput({ target: { value: '' } });
     };
 
+    onCompositionStart = () => {
```

Comment:
Looks like the convention in this file is `handle` rather than `on`, so `handleCompositionStart`.